### PR TITLE
Docker Support for OSX

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by inputfail** [comment]
"Approved" as functionality was tested on macOS 10.14 :)

File: fixbaserom.py [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
+            sys.exit(0)
+
 # Determine if we have a ROM file
 romFileName = ""
-if (path.exists("baserom_original.z64")):
+if path.exists("baserom_original.z64"):
     romFileName = "baserom_original.z64"
-elif (path.exists("baserom_original.n64")):
+elif path.exists("baserom_original.n64"):
     romFileName = "baserom_original.n64"
```
Comment (fig02) [inline fixbaserom.py:24]: still think we should go with input.z64 to clear up the confusion about the meaning of original. readme and error message below would need updating as well

Comment (ethteck) [reply fixbaserom.py:24]: oh right. one sec

Comment (Roman971) [reply fixbaserom.py:24]: input is more confusing than original imo.

Comment (ethteck) [reply fixbaserom.py]: As far as I can tell, the readme doesn't mention baserom_original - did you mean the wiki?

**Pull request merged**